projects/ngx-datatable/src/lib/components/body/body.component.ts

```diff
@@ -340,6 +340,15 @@ export class DataTableBodyComponent<TRow extends Row = any> implements OnInit, O
     return undefined;
   });
 
+  readonly detailRowHeightFn = computed(() => {
```

Comment:
@spike-rabbit it seems something went wrong with the rebase: You now add this function which is a copy of `getDetailRowHeight` while `getDetailRowHeight` is still present in the codebase yet unused.